clustering/Clustering_map.R

```diff
@@ -0,0 +1,308 @@
+# Load libaries 
+library(tidyverse) 
+library(dplyr)
+library(readxl)
+library(ggplot2)
+library(openxlsx)
+library(sf)
+
+# Set seed
+set.seed(123)
+
+# Get Windows username
+win_username <- Sys.getenv("USERNAME")
+
+# Load in data
+file_name = "Cluster_Maps_Data_20240221"
+glob_df <- read_excel(paste0("C:/Users/",win_username,"/Office for National Statistics/Strategy, Coordination and Dissemination - Documents/Sharepoint_data_loading/Clustering/Outputs/Clustering_model_outputs/",file_name,".xlsx"), 
```

Comment:
This will need to be removed as contains a sharepoint path. Maybe make a variable for path and put that in here instead